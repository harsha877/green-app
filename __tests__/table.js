import React from 'react';
import { render} from '@testing-library/react-native';
import Table from '../src/components/table'

test("Table Component Loading",()=>{
    render(<Table data ={{}} onPressHandler={()=>{}}/>)
 
 })
 
 test("Table component snapshot",()=>{
    let TableComponent=render(<Table data ={{}} onPressHandler={()=>{}}/>).toJSON();
    expect(TableComponent).toMatchSnapshot();
 })

 