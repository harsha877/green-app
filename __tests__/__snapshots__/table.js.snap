// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table component snapshot 1`] = `
<RCTScrollView
  data={Object {}}
  getItem={[Function]}
  getItemCount={[Function]}
  keyExtractor={[Function]}
  nestedScrollEnabled={true}
  onContentSizeChange={[Function]}
  onLayout={[Function]}
  onMomentumScrollBegin={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={Array []}
  viewabilityConfigCallbackPairs={Array []}
>
  <View />
</RCTScrollView>
`;
